packages/web-pkg/src/components/Spaces/QuotaModal.vue

```diff
@@ -181,15 +182,20 @@ export default defineComponent({
           value: driveData.quota
         })
       })
-      const results = await Promise.allSettled<Array<unknown>>(requests)
+      const results = await loadingService.addTask(() => {
+        return Promise.allSettled<Array<unknown>>(requests)
+      })
       const succeeded = results.filter((r) => r.status === 'fulfilled')
       if (succeeded.length) {
         store.dispatch('showMessage', { title: getSuccessMessage(succeeded.length) })
       }
       const errors = results.filter((r) => r.status === 'rejected')
       if (errors.length) {
         errors.forEach(console.error)
-        store.dispatch('showErrorMessage', { title: getErrorMessage(errors.length) })
+        store.dispatch('showErrorMessage', {
+          title: getErrorMessage(errors.length),
+          errors
```

Comment:
In the other files you add `.map((f) => f.reason)` to `errors`. Why is it not necessary here?